integration/tests/api/docker_test.go

```diff
@@ -253,6 +253,9 @@ func TestDockerContainerNetworkStats(t *testing.T) {
 	containerId := fm.Docker().RunBusybox("watch", "-n1", "wget", "https://www.google.com/")
 	waitForContainer(containerId, fm)
 
+	// Add a little sleep to let the request complete a few times
```

Comment:
The `wget` req on line 253 (https://github.com/google/cadvisor/pull/993/files#diff-6333e565f1256ee66767acf24186c52cR253). That's what should be causing network traffic. The problem I think that's happening is there's a race between container is up check via `waitForContainer` & first `wget` requests.